topics/elevate-resources.md

```diff
@@ -32,5 +32,6 @@ body_class: elevate-resources
 | `month_date_created` | `INTEGER` | Month of date created | `12` |
 | `week_date_created` | `INTEGER` | ISO 8601 week, note Monday is the start of the ISO 8601 week, values 1-53 | `52` |
 | `dow_date_created` | `INTEGER` | Day of week, values 0-6, Sunday is 0 | `5` |
+| `isbn` | `VARCHAR(13)`  | The ISBN of the resource | `978097982900` |
```

Comment:
It might be worth clarifying here that this will only be set for SAGE Catalyst textbooks or textbook chapters that were elevated, just to avoid any confusion that it might also be set for resources created via Copyright Clearance.

Reply:
I've updated the description, let me know what you think.

Reply:
Looks good. If @timhodson is happy for this to go in then so am I.